### DISCUSSION TITLE
I have not received my education benefits within 5 days of my approval status

### DISCUSSION THREAD

**Discussion by Harvey34299** [community/community#129266] [GitHub Education]
### Hi everyone,

I was approved on June 17th, and it has been 4 days by now (June 21). But I still haven't received the education benefits. Could the education community team solve this problem for me?
![2024-06-21 111553](https://github.com/community/community/assets/93419163/1a95304c-f611-48e2-8292-b9c0eb77623e)

**Comment by mdjeghbala** [comment]
# It can take time sometimes, probably because there are so many requests.

![Screenshot_1](https://github.com/community/community/assets/115470403/67d65ddf-d18a-4648-bb86-10abc2846c34)

You can see this in this link: https://support.github.com/request/education
